src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsFromStaticMethodRefExtended.java

```diff
@@ -0,0 +1,34 @@
+/*
+UnusedImports
+processJavadoc = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.util.Optional; // ok
+
+import java.util.function.Function; // ok
+import java.util.List; // ok
+import java.util.Arrays; // ok
```

Comment:
Please do me favor to remove all `ok` comments that you introduce.
Trailing comment should be red flag of attention to reader.
If no error, should be no attention.

Reply:
Fixed.